net/netfilter/nf_tables_api.c

```diff
@@ -9012,13 +9041,16 @@ static void __nft_release_tables(struct net *net)
 
 static int __net_init nf_tables_init_net(struct net *net)
 {
+	struct nftables_pernet *nft_net = nft_pernet(net);
+
 	INIT_LIST_HEAD(&net->nft.tables);
 	INIT_LIST_HEAD(&net->nft.commit_list);
 	INIT_LIST_HEAD(&net->nft_module_list);
 	INIT_LIST_HEAD(&net->nft_notify_list);
 	mutex_init(&net->nft_commit_mutex);
 	net->nft.base_seq = 1;
 	net->nft.validate_state = NFT_VALIDATE_SKIP;
+	nft_net->gc_seq = 0;
```

Comment:
This should be a part of `netfilter: nf_tables: GC transaction API to avoid race with control plane`
https://github.com/torvalds/linux/commit/5f68718b34a531a556f2f50300ead2862278da26#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR11140

Reply:
Fixed while fixing something else, it's belonging to the right commit now.